packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -146,26 +146,17 @@ const ElementHook: Hooks<VElement> = {
     },
 };
 
-function isUpgradableComponent(
```

Comment:
not needed anymore since we don't care if it is upgrable or not, we just pass the optional first argument to the constructor.